docs/readmes/resources/pcap.md

```diff
@@ -0,0 +1,55 @@
+---
+id: pcap
+title: PCAP Collection
+hide_title: true
+---
+
+*Last Updated: 4/20/2021*
```

Comment:
For both files: these timestamps aren't necessary / won't get updated when people make changes. Docusaurus has a feature to add these automatically, so look into that if you're interested in providing this affordance